File: version_java/src/main/java/com/example/version_java/ui/adapter/viewholder/CriminalViewHolder.java
package com.example.version_java.ui.adapter.viewholder;

import android.util.Log;

import androidx.recyclerview.widget.RecyclerView;

import com.example.version_java.data.model.DistanceCriminal;
import com.example.version_java.databinding.ItemCriminalListBinding;

import org.jetbrains.annotations.NotNull;

public final class CriminalViewHolder extends RecyclerView.ViewHolder {
    private final ItemCriminalListBinding binding;

    public final void bind(@NotNull DistanceCriminal item) {
        Log.d("결과", item.getName());
        this.binding.setItem(item);
    }

    public CriminalViewHolder(@NotNull ItemCriminalListBinding binding) {
        super(binding.getRoot());
        this.binding = binding;
    }
}

File: app/src/main/java/com/example/crimenotification/ui/adapter/viewholder/CriminalViewHolder.kt
package com.example.crimenotification.ui.adapter.viewholder

import android.view.LayoutInflater
import android.view.ViewGroup
import androidx.recyclerview.widget.RecyclerView
import com.example.crimenotification.R
import com.example.crimenotification.data.model.DistanceCriminal
import com.example.crimenotification.databinding.ItemCriminalListBinding

class CriminalViewHolder(parent: ViewGroup) : RecyclerView.ViewHolder(
    LayoutInflater.from(parent.context).inflate(R.layout.item_criminal_list, parent, false)
) {
    private val binding = ItemCriminalListBinding.bind(itemView)

    fun bind(
        item: DistanceCriminal
    ) {
        binding.item = item
    }
}